TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.CfgSectionSymbol.cs

```diff
@@ -0,0 +1,42 @@
+﻿using TypeCobol.Compiler.Scopes;
+using TypeCobol.Compiler.Symbols;
+
+namespace TypeCobol.Analysis.Cfg
+{
+    public partial class ControlFlowGraphBuilder<D>
+    {
+        /// <summary>
+        /// A Section symbol used by Cfg : it contains Sentences and Paragraphs in order of appearance.
+        /// </summary>
+        internal class CfgSectionSymbol : SectionSymbol
+        {
+            /// <summary>
+            /// Constructor
+            /// </summary>
+            /// <param name="name">Section's name</param>
+            public CfgSectionSymbol(string name) : base(name)
+            {
+                SentencesParagraphs = new Domain<Symbol>(this);
+            }
+
+            /// <summary>
+            /// All sentences and paragraphs in this section in the order of appearance.
+            /// </summary>
+            public Domain<Symbol> SentencesParagraphs
```

Comment:
Add a method to yield sentences from a section. Users of the classs would not have to test symbol kind.

Reply:
Ok but the same thing must be done for CfgParagraphSymbol, as you suggest later it will be necessay to have a common base class for CfgSectionSymbol and CfgParagraphSymbol and use code in method 
```C#
private IEnumerable<CfgSentence> YieldSectionOrParagraphSentences(Symbol sectionOrParagraphSymbol)
```
The problem is that CfgSectionSymbol must ihnerits SectionSymbol from SDLite and CfgParagraphSymbol must inherits ParagraphSymbol from SDLite, thus we should create an Interface like ISentenceContainer that implements such contract.

Reply:
See PR #1738.